.circleci/config.yml

```diff
@@ -37,6 +37,9 @@ parameters:
   create-release:
     type: boolean
     default: false
+  cache-version:
+    type: string
+    default: 'dependency-cache-v3-'
```

Comment:
Decided to change this into a pipeline parameter so we can clear the cache with an api call as needed.